protocol/p2p/src/core/hub.rs

```diff
@@ -138,6 +138,11 @@ impl Hub {
         self.peers.read().values().map(|r| r.as_ref().into()).collect()
     }
 
+    /// Returns the number of currently active peers
+    pub fn active_peers_len(&self) -> usize {
+        self.peers.read().values().len()
```

Comment:
no need for `.values()` here, just use len directly over the map

Reply:
fixed in https://github.com/aspectron/rusty-kaspa/commit/1ee65c349a9e663878787973d73ae2d4b420f44d